CONTRIBUTING.md

```diff
@@ -273,9 +273,13 @@ If you add a new general flag (not for a backend), then document it in
 alphabetical order.
 
 If you add a new backend option/flag, then it should be documented in
-the source file in the `Help:` field.  The first line of this is used
-for the flag help, the remainder is shown to the user in `rclone
-config` and is added to the docs with `make backenddocs`.
+the source file in the `Help:` field. The first line must be a single
```

Comment:
Should probably mention that it shouldn't be too long - ideally <80 characters. If it is too long the text wraps too many times in a small (or large!) terminal.